README.md

```diff
@@ -9,85 +9,117 @@ website: cltl.nl
 https://github.com/cltl/StoryTeller
 
 ## DESCRIPTION of the StoryTeller stack
-The Storyteller system will allow you to generate general queries about topics on the [KnowledgeStore](https://knowledgestore.fbk.eu/) and visualize the results for analysis and exploration. It tries to accomplish this task by providing the user with web frontends that simplify the information needed for the complex tasks required and allowing the user to explore the knowledge with custom selections.
+
+The Storyteller system will allow you to generate general queries about topics
+on the [KnowledgeStore](https://knowledgestore.fbk.eu/) and visualize the
+results for analysis and exploration. It tries to accomplish this task by
+providing the user with web frontends that simplify the information needed for
+the complex tasks required and allowing the user to explore the knowledge with
+custom selections.
 
 ## Installation
 ### Automated installation of the complete query building and visualization suite
+
 (please make sure to install docker and docker-compose https://www.docker.com/)  
 
-After installation of docker, the following bash commands should suffice to install the entire system:
+After installation of docker, the following bash commands should suffice to
+install the entire system:
 
 ```bash
-    docker volume create --name=data
-    docker-compose up
+docker volume create --name=data
+docker-compose up
 ```
-This will create a set of docker containers that will run the following system for you:  
-![diagram of the stack](https://cdn.rawgit.com/NLeSC-Storyteller/StoryTeller/master/doc/stack-diagram.svg "Stack diagram")  
+This will create a set of docker containers that will run the following system
```

Comment:
diagram is accurate but has no end or beginning/ no logical order, makes it harder to read.

perhaps a better way is to present the information incrementally. I would break it down into 3 topics: 

1. what the user sees: the two frontends (_StoryTeller_ and _query-builder-client_) 
2. what information source the user queries (_Knowledgestore overview_, _queries_), and how the query works (_querybuilder-server_)
3. how _Knowledgestore overview_ is filled with useful data.

introduce these step by step